### PR TITLE
khalid/scroll to error message

### PR DESCRIPTION
**Pull request by Khalidxx**
When a new account cannot be opened, the view is automatically scrolled to the error message

### CODE REVIEW

File: src/javascript/app/common/account_opening.js [hunk 1]
```diff
@@ -197,9 +197,11 @@ const AccountOpening = (() => {
     const handleNewAccount = (response, message_type) => {
         if (response.error) {
             const errorMessage = response.error.message;
+            const noticeBox    = $('#client_message').find('.notice-msg');
```
Comment (negarn) [inline src/javascript/app/common/account_opening.js]: for jQuery elements we use `$` in front of the name to make it obvious, and also the format should be lowercase letters separated by `_`, so `$notice_box`. could you also fix `error_message` above that? seems it was missed

File: src/javascript/app/common/account_opening.js [hunk 2]
```diff
@@ -197,9 +197,11 @@ const AccountOpening = (() => {
...
             $('#submit-message').empty();
-            $('#client_message').find('.notice-msg').text(response.msg_type === 'sanity_check' ? localize('There was some invalid character in an input field.') : errorMessage).end()
+            noticeBox.text(response.msg_type === 'sanity_check' ? localize('There was some invalid character in an input field.') : errorMessage).end()
                 .setVisibility(1);
+            $.scrollTo(noticeBox, 500, { offset: -150 });
```
Comment (negarn) [inline src/javascript/app/common/account_opening.js]: why such a large offset?

File: src/javascript/app/common/account_opening.js (hunk 1 above)
Comment (Khalidxx) [reply src/javascript/app/common/account_opening.js]: Roger!

File: src/javascript/app/common/account_opening.js (hunk 2 above)
Comment (Khalidxx) [reply src/javascript/app/common/account_opening.js]: It scrolls so as to keep the message at the top of the view. I was trying to get the message in the middle

**Pull request merged**